tools/pipeline-owners-extractor/Azure.Sdk.Tools.PipelineOwnersExtractor/Azure.Sdk.Tools.PipelineOwnersExtractor.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <PackAsTool>true</PackAsTool>
+    <ToolCommandName>pipeline-codeowner-extractor</ToolCommandName>
```

Comment:
Do you want the tool name to match up with the project name?

Reply:
I do

Reply:
Should it be `pipeline-owners-extractor` then?

Reply:
it should :)